Mathlib/LinearAlgebra/TensorProduct.lean

```diff
@@ -1016,6 +1025,16 @@ theorem rTensor_tmul (m : M) (n : N) : f.rTensor M (n ⊗ₜ m) = f n ⊗ₜ m :
   rfl
 #align linear_map.rtensor_tmul LinearMap.rTensor_tmul
 
+lemma comm_comp_rTensor_comp_comm_eq {g : N →ₗ[R] P} :
+    (TensorProduct.comm R P Q) ∘ₗ (rTensor Q g) ∘ₗ (TensorProduct.comm R Q N) =
+      lTensor Q g :=
+  TensorProduct.ext rfl
+
+lemma comm_comp_lTensor_comp_comm_eq {g : N →ₗ[R] P} :
+    (TensorProduct.comm R Q P) ∘ₗ (lTensor Q g) ∘ₗ (TensorProduct.comm R N Q) =
```

Comment:
a nit: we generally don't write `∘ₗ` (or indeed any operator) with parens around its operands
```suggestion
    TensorProduct.comm R Q P ∘ₗ lTensor Q g ∘ₗ TensorProduct.comm R N Q =
```

Reply:
OK